### PR TITLE
Problem: potentially vulnerable dependencies

### PR DESCRIPTION
**Pull request by tomtau**
Solution: updated ibc-go, Tendermint and Go compiler to the patched versions

### CODE REVIEW

**Review by yihuang — APPROVED** [review]
LGTM

**Review by devashishdxt — APPROVED** [review]

File: app/test_helpers.go [hunk 1]
```diff
@@ -380,6 +382,7 @@
 	var err error
 	for i := 0; i < numToGenerate; i++ {
 		txs[i], err = helpers.GenTx(
+			rand.New(rand.NewSource(time.Now().UnixNano())),
```
[the remark this answers is not in the document]
Comment (tomtau) [reply app/test_helpers.go:385]: used in tests

File: app/test_helpers.go [hunk 2]
```diff
@@ -330,6 +331,7 @@
 ) (sdk.GasInfo, *sdk.Result, error) {
 
 	tx, err := helpers.GenTx(
+		rand.New(rand.NewSource(time.Now().UnixNano())),
```
[the remark this answers is not in the document]
Comment (tomtau) [reply app/test_helpers.go:334]: used in tests

File: go.mod [hunk 3]
```diff
@@ -146,8 +146,8 @@ replace github.com/opencontainers/image-spec => github.com/opencontainers/image-
 replace github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.3
 
 // needed due to a breaking change in 0.44.6
-replace github.com/cosmos/cosmos-sdk => github.com/crypto-org-chain/cosmos-sdk v0.44.8-patch-dragonberry
+replace github.com/cosmos/cosmos-sdk => github.com/crypto-org-chain/cosmos-sdk v0.44.8-patch5
```
Comment (tomtau) [inline go.mod:149]: https://github.com/crypto-org-chain/cosmos-sdk/compare/v0.44.8-patch-dragonberry...v0.44.8-patch5

**Pull request merged**